From c14bac4f216c452d1e82cb73e4b53c2ccdd77987 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Tue, 2 Oct 2012 22:19:58 -0400
Subject: [PATCH] version bump to 2.0

This is somewhat redundant as most distros are rolling snapshots
themselves, but maybe we should celebrate by throwing a 2.0 release
out there.  It's been over 10 years since the last release after all.

Note: We'll need Bernd to actually run `make dist` and post the tarball
	to sf.net.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b2ea629..155e249 100644
--- a/Makefile
+++ b/Makefile
@@ -102,7 +102,7 @@ endif
 # -------- end of user definitions --------
 
 MAINTAINER = Philip.Blundell@pobox.com
-RELEASE	   = 1.60
+RELEASE	   = 2.0
 
 .EXPORT_ALL_VARIABLES:
 
-- 
1.7.11.4

